tasks/run.js

```diff
@@ -78,6 +78,7 @@ function runLocal(opts) {
 		}
 
 		if(opts.nodemon) {
+			args.push('--ignore', 'public/');
```

Comment:
Most apps' `nodemon.json` includes `public/head.css` do we want to include that or just ignore `/public` completely? It seemed to only be `public/about.json` that caused all the restarts